spec/controllers/aspect_memberships_controller_spec.rb

```diff
@@ -69,6 +70,7 @@
       post :create, params
       post :create, params
       expect(response.status).to eq(400)
+      expect(response.body).to include(I18n.t('aspect_memberships.destroy.invalid_statement'))
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.